examples/generate.sh

```diff
@@ -83,8 +83,23 @@ else
   exit 1
 fi
 
+# Check if correct controlplane preference is given
 if [[ -n "${3-}" ]] && [[ $3 != -* ]] && [[ $3 != --* ]]; then
-  OUTPUT_DIR=$(echo $3 | tr '[:upper:]' '[:lower:]')
+  if [[ "$3" == "single-node" || "$3" == "multi-node" ]]; then
+    export CONTROLPLANE=$3
+  else"$
```

Comment:
```suggestion
  else
```

I think you have to remove the "$

Reply:
My bad. Will update